server/model.js

```diff
@@ -0,0 +1,109 @@
+const pool = require('./db');
+
+module.exports = {
+  getAllReviews: async (product_id, count, page) => {
+    const limit = count;
+    const offset = (page === undefined || page <= 1) ? 0 : (page - 1) * count;
+    const queryParams = [product_id, offset, limit];
+    const queryString = `SELECT
+         r.id,
+         r.rating,
+         r.summary,
+         r.recommend,
+         r.response,
+         r.body,
+         r.date,
+         r.reviewer_name,
+         r.helpfulness,
+         (SELECT coalesce(json_agg(
+                json_build_object(
+                    'id', id,
+                    'url', url
+                    )), '[]'::json)
+                AS Photos FROM photos WHERE r.id = review_id
+          )
+      FROM reviews r
+      WHERE r.product_id = $1 OFFSET $2 LIMIT $3`;
+    try {
+      return await pool.query(queryString, queryParams);
```

Comment:
From the [node-postgres docs on pooling](https://node-postgres.com/features/pooling), I think you would need to do:

1. `pool.connect()` to checkout a client from the pool
2. `client.query()` to run your query using the client
3. `client.release()` to return your client back into the pool.